modules/cache.js

```diff
@@ -18,7 +18,7 @@ var async = require('async');
 var transactionTypes = require('../helpers/transaction_types.js');
 
 var cacheReady = true;
-var errorCacheDisabled = 'Cache Unavailable';
+var errorCacheDisabled = 'Cache disabled';
```

Comment:
`Cache disabled` here, and `Cache Disabled` in app.js:627L. Please make unify the case - `Cache Disable in both places`.